README.md

```diff
@@ -41,14 +41,16 @@ jobs:
         working-directory: checkout
 ```
 
-[.github/workflows/applanga-pull.yml](https://github.com/applanga/github-workflow-example/blob/master/.github/workflows/applanga-pull.yml) pulls new translations available from Applanga and then creates a pull request in the repo with the newly added languages or updated translation files. For this to work a **Webhook Endpoint** has to be configured for the project on Applanga dashboard to trigger the workflow. See [Configure Webhook Endpoint](#configure-webhook-endpoint) below for how to configure 
+Note that for the above workflow file the `Applanga CLI` will only get triggered when there is a push to the `master` branch as specified in the config. It is also possible to specify a different branch or add more branches as needed.
+
+[.github/workflows/applanga-pull.yml](https://github.com/applanga/github-workflow-example/blob/master/.github/workflows/applanga-pull.yml) pulls new translations available from Applanga and then creates a pull request in the repo with the newly added languages or updated translation files. The workflow configuration is configured to run on any given branch. To set this up successfully there are 2 important requirements
+1. The workflow file must first be created from the default branch. In other words start first by adding the workflow file to the `master` branch of the repository, which allows github to pick up the workflow. This is a github limitation, and more info can be found on the following github [issue](https://github.community/t/workflow-dispatch-event-not-working/128856/2)
```

Comment:
please do not use 'master' use 'main' if needed or better 'default branch'

Reply:
changed